docs/index.md

```diff
@@ -1,17 +1,18 @@
 ---
 layout: page
-title: AddressBook Level-3
+title: NUS ModBook
 ---
 
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
```

Comment:
this needs to be updated as well

Reply:
where do we get the badge from though? couldn't find it in our project files

Reply:
u can get it from the github action workflow page!